packages/rollup-plugin-import-meta-assets/test/fixtures/backticks.js

```diff
@@ -0,0 +1,4 @@
+const backticksImg = new URL(`./one.svg`, import.meta.url);
+const backticksImg2 = new URL(`./two.svg`, import.meta.url);
```

Comment:
I'd add a negative scenario case with a template literal and check in the test that it didn't change

Reply:
Hmm I see your point, currently I guess it ignores it and leaves such statements as is. I've added it to the test.

I was actually thinking of raising another PR that detects template literals with expressions in new URL() or other cases of dynamic-ness that cannot be statically analysed. In those cases I **think** the plugin should throw with a helpful error message explaining what's wrong... Why throw? Because it's a bit strange I think to just ignore it and have a path that's untransformed and most definitely pointing to a non-existent file in the final bundle, leading to a 404 error, so it's better to error earlier and make it clear what's wrong. And there's always warnOnError option in the plugin which would then cause the current behavior, where it's just ignored and untransformed (except with a nice warning). 
I imagine that's a breaking change though? Thoughts?

Reply:
Should also consider btw that maybe it should not throw, because if you use it in combination with @rollup/plugin-dynamic-import-vars, such paths might actually work..